app/controllers/api/v0/base_controller.rb

```diff
@@ -20,7 +20,7 @@ def list_portfolios
       end
 
       def list_portfolio_items
-        render :json => PortfolioItem.all
+        render :json => PortfolioItem.kept
```

Comment:
Now that we have a default scope of `kept` is this change needed?  It is more explicit so I'm not against it.

Reply:
I agree with @gmcculloug - We don't need `.kept` anymore but I like that it is more explicit.

Reply:
I *think* this should fall out since it's one of the commits from the `portfolio_items` branch (ie #120 ). I updated it there, but the early commit where I was using kept. So the change is there and after it's merged it shouldn't show up here.